src/components/countries/countries.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import CountriesList from './CountriesList';
+
+const Countries = ({ countries }) => {
+  const [input, setInput] = useState('');
+
+  const filteredData = countries.filter((country) => {
+    const data = Object.keys(country).some((key) => {
+      const res = country[key].toString().toLowerCase().includes(input.toString().toLowerCase());
+      return res;
+    });
+    return data;
+  });
+
+  return (
+    <div className="section">
+      <div className="input">
+        <input
+          type="search"
+          className="search"
+          placeholder="Search by country name .."
+          value={input}
+          onChange={(e) => setInput(e.target.value)}
+          required
```

Comment:
**OPTIONAL** - To help improve the UI/ UX of your site, I suggest you display a message when a user tries to search for a country that's not on the list. Please fix this 🙏